test/integration/operator/garden/garden_test.go

```diff
@@ -204,7 +204,7 @@ var _ = Describe("Garden controller tests", func() {
 				},
 				VirtualCluster: operatorv1alpha1.VirtualCluster{
 					DNS: operatorv1alpha1.DNS{
-						Domain: "virtual-garden.local.gardener.cloud",
+						Domain: pointer.String("virtual-garden.local.gardener.cloud"),
```

Comment:
Same as other tests?
```suggestion
						Domains: []string{"virtual-garden.local.gardener.cloud"},
```